site/ipam.md

```diff
@@ -22,7 +22,35 @@ You can see which address was allocated with
     host1# weave ps $C
     a7aff7249393 7a:51:d1:09:21:78 10.2.3.1/24
 
-The `-iprange` parameter is given in [CIDR
+You may wish to [run containers on different
+subnets](features.html#application-isolation). To set a default
+subnet, launch with `-ipsubnet`:
+
+    host1# weave launch -iprange 10.2.0.0/16 -ipsubnet 10.2.3.0/24
+
+`-iprange` should cover the entire range that you will ever use for
+allocation, and `-ipsubnet` is the subnet that will be used when
+you don't explicitly specify one.
+
+Then, to request the allocation of an address from a subnet other than
+the default, use `--subnet`:
+
+    host1# C=$(weave run --subnet 10.2.7.0/24 -ti ubuntu)
+
+To explicitly request the default subnet, use `--subnet default`.
+
+Note that `-iprange` can be smaller than `-ipsubnet`, if you want to
+use a mixture of automatically-allocated addresses and manually-chosen
+addresses, and have the containers communicate with each other. For
+example, if you say:
+
+    host1# weave launch -iprange 10.9.0.0/17 -ipsubnet 10.9.0.0/16
+
+then you can run all containers in the 10.9.0.0/16 subnet, with IPAM
+using the lower half and leaving the upper half free for manual
+allocation.
+
+The subnet parameters are written in [CIDR
```

Comment:
right. Though I have to say that to me it's not at all obvious which is which when looking at something like

    weave run 10.2.0.1/24 --subnet 10.2.1.0/24 10.2.2.1/24

We could use a prefix instead, i.e. `subnet:10.2.1.0/24`